packages/electrode-archetype-react-app/arch-clap.js

```diff
@@ -754,24 +754,27 @@ Individual .babelrc files were generated for you in src/client and src/server
     "test-frontend-ci": ["?.karma.test-frontend-ci"],
 
     ".jest.test-frontend-cov": () => {
+      let runJest;
       const testDir = ["_test_", "__tests__"].find(x => shell.test("-d", x));
-
-      const srcJestFiles =
-        !testDir &&
-        glob.sync(`**/*.{test,spec}.{js,jsx,ts,tsx}`, {
+      if (! testDir) {
```

Comment:
There are three scenarios that we want to run jest test.
First: If user has a directory that's named `_test_` or `__tests__`.   
Second: If any files name matches `*.{test,spec}.{js,jsx,ts,tsx}`
Third: Users has their own config to identify tests, in which case we just run jest.

Reply:
Note that if **any** of the scenarios is true independently, we run jest test.